Patch104pZH/GameFilesEdited/Data/INI/Weapon.ini

```
@@ -5878,7 +5892,7 @@ End
 ;  DamageType                  = COMANCHE_VULCAN  ;used only for this weapon so stinger sites don't lose their guys but otherwise acts just like Small_Arms
 ;  DeathType                   = NORMAL
 ;  WeaponSpeed                 = 999999.0          ; dist/sec (huge value == effectively instant)
-;  ProjectileObject            = DummyWeaponProjectile ; Patch104p @bugfix from NONE
+;  ProjectileObject            = DummyWeaponProjectile
 ;  RadiusDamageAffects         = ALLIES ENEMIES NEUTRALS
 ;  DelayBetweenShots           = 100         ; time between shots, msec
```

```suggestion
;  DelayBetweenShots = 999999 ; time between shots, msec
```
We add `DummyWeaponProjectile`, may as well do this.

---

fixed.